File: Basics/0.3-Conditional-Statements-Advanced/Exercise/operations_between_numbers.py
n1 = int(input())
n2 = int(input())
operator = input()
result = None
is_even = "even"

# if operator == "+":
#     result = n1 + n2
#     if result % 2 == 0:
#         print(f"{n1} {operator} {n2} = {result} - even")
#     else:
#         print(f"{n1} {operator} {n2} = {result} - odd")
# elif operator == "-":
#     result = n1 - n2
#     if result % 2 == 0:
#         print(f"{n1} {operator} {n2} = {result} - even")
#     else:
#         print(f"{n1} {operator} {n2} = {result} - odd")
# elif operator == "*":
#     result = n1 * n2
#     if result % 2 == 0:
#         print(f"{n1} {operator} {n2} = {result} - even")
#     else:
#         print(f"{n1} {operator} {n2} = {result} - odd")
if operator == "+" or operator == "-" or operator == "*":
    if operator == "+":
        result = n1 + n2
    elif operator == "-":
        result = n1 - n2
    elif operator == "*":
        result = n1 * n2

    if result % 2 == 0:
        print(f"{n1} {operator} {n2} = {result} - even")
    else:
        print(f"{n1} {operator} {n2} = {result} - odd")
elif operator == "/":
    if n1 == 0 or n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        result = n1 / n2
        print(f"{n1} / {n2} = {result:.2f}")
elif operator == "%":
    if n1 == 0 or n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        result = n1 % n2
        print(f"{n1} % {n2} = {result}")

File: JS-Basics/0-3-Conditional-Steatements-Advanced/Exercise/operations-between-numbers.js
function calculator(input) {
    let n1 = Number(input[0]);
    let n2 = Number(input[1]);
    let operator = input[2];
    let result;
    if (operator == "+") {
        result = n1 + n2;
    } else if (operator == "-") {
        result = n1 - n2;
    } else if (operator == "*") {
        result = n1 * n2;
    } else if (operator == "/") {
        if (n2 == 0) {
            console.log(`Cannot divide ${n1} by zero`);
            return;
        } else {
            result = n1 / n2;
            console.log(`${n1} / ${n2} = ${result.toFixed(2)}`);
            return;
        }
    } else if (operator == "%") {
        if (n2 == 0) {
            console.log(`Cannot divide ${n1} by zero`);
            return;
        } else {
            result = n1 % n2;
            console.log(`${n1} % ${n2} = ${result}`);
            return;
        }
    }

    let numType;
    if (result % 2 == 0) {
        numType = "even";
    } else {
        numType = "odd";
    }
    console.log(`${n1} ${operator} ${n2} = ${result} - ${numType}`)
}